funda/spiders/funda_spider.py

```diff
@@ -1,18 +1,16 @@
 import re
 import scrapy
-from scrapy.spiders import CrawlSpider, Rule
 from scrapy.linkextractors import LinkExtractor
 from funda.items import FundaItem
+import logging
 
-class FundaSpider(CrawlSpider):
+class FundaSpider(scrapy.Spider):
 
     name = "funda_spider"
     allowed_domains = ["funda.nl"]
-
-    def __init__(self, place='amsterdam'):
-        self.start_urls = ["http://www.funda.nl/koop/%s/p%s/" % (place, page_number) for page_number in range(1,301)]
-        self.base_url = "http://www.funda.nl/koop/%s/" % place
-        self.le1 = LinkExtractor(allow=r'%s+(huis|appartement)-\d{8}' % self.base_url)
+    start_urls = ["https://www.funda.nl/koop/amsterdam/p%s/" % (page_number) for page_number in range(500)]
```

Comment:
```suggestion
    def __init__(self, place= 'amsterdam') -> None:
        self.start_urls = [f"https://www.funda.nl/koop/{place}/p{page_number}/"  for page_number in range(500)]
        self.base_url = f"https://www.funda.nl/koop/{place}/"
        self.le1 = LinkExtractor(allow=r'%s+(huis|appartement)-\d{8}' % self.base_url)
```

replace % string with f-string and make place a variable again.
This should still work